docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component/Feature
+     - Status
+     - Deprecated on
+   * - :doc:`Ledger API specification </app-dev/grpc/proto-docs>` including all semantics of >= DAML-LF 1.6
+     - Stable
+     -   
+   * - CLI and test names of :doc:`/tools/ledger-api-test-tool/index`
+     - Beta
+     -
+   * - `Numbered (ie non-dev) Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
+     - Stable
+     - 
+   * - `Dev Versions of Proto definitions distributed via GitHub Releases <protobufs_>`_
+     - Alpha
+     - 
+
+.. _runtime-components:
+
+Runtime components
+------------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **JSON API**
+     - 
+     -
+   * - HTTP endpoints under ``/v1/`` including status codes, authentication, query language and encoding.
+     - Stable
+     -
+   * - ``daml json-api`` CLI :ref:`for development <dev-tools>`. (as specified using ``daml json-api --help``)
+     - Stable
+     -
+   * - Stand-alone distribution for production use, including CLI specified in ``--help``.
+     - Stable
+     -
+   * - **Triggers**
+     -
+     -
+   * - :doc:`DAML API of individual Triggers </triggers/index>`
+     - Beta
+     -
+   * - :doc:`Development CLI to start individual triggers in dev environment </triggers/index>` (``daml trigger``)
+     - Beta
+     -
+   * - :doc:`/tools/trigger-service` (``daml trigger-service``)
+     - Alpha
+     -
+   * - **Extractor**
+     -
+     -
+   * - :doc:`/tools/extractor` (``daml extractor``)
+     - Labs
+     -
+
+.. _libraries:
+
+Libraries
+---------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **Scala Ledger API Bindings**
+     -
+     -
+   * - ``daml codegen scala`` :doc:`CLI and generated code </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - ``bindings-scala_2.12`` :doc:`library and its public API </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - **Java Ledger API Bindings**
+     - 
+     -
+   * - ``daml codegen java`` :doc:`CLI and generated code </app-dev/bindings-java/codegen>`
+     - Stable
+     -
+   * - ``bindings-java`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.6-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.7-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.8-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-dev-archive-java-proto``
+     - Alpha
+     -
+   * - **Node.js Ledger API Bindings**
+     -
+     -
+   * - ``@digital-asset/binings-js`` :doc:`Node.js library </app-dev/bindings-js>`
```

Comment:
```suggestion
   * - ``@digital-asset/bindings-js`` :doc:`Node.js library </app-dev/bindings-js>`
```

Reply:
The readme for those bindings currently states that they are experimental, Marking them as stable here only to then deprecate them seems weird.

Reply:
I did not realize the readme said that. That readme also isn't linked from our docs so I'm not sure whether this was discoverable enough. Calling them "Beta" or "Alpha" now after having them in our docs outside of "Early Access" for a long time would seem underhanded.